platforms/jvm/ear/src/test/groovy/org/gradle/plugins/ear/EarTest.groovy

```diff
@@ -21,13 +21,30 @@ import org.gradle.api.file.CopySpec
 import org.gradle.api.plugins.JavaBasePlugin
 import org.gradle.api.tasks.bundling.AbstractArchiveTask
 import org.gradle.api.tasks.bundling.AbstractArchiveTaskTest
+import org.gradle.internal.file.PathToFileResolver
 import org.gradle.plugins.ear.descriptor.DeploymentDescriptor
 import org.gradle.plugins.ear.descriptor.EarSecurityRole
 import org.gradle.plugins.ear.descriptor.internal.DefaultDeploymentDescriptor
 import org.gradle.test.fixtures.archive.JarTestFixture
 
 class EarTest extends AbstractArchiveTaskTest {
     Ear ear
+    PathToFileResolver nullPathToFileResolver = new PathToFileResolver() {
+        @Override
+        File resolve(Object path) {
+            return null
+        }
+
+        @Override
+        PathToFileResolver newResolver(File baseDir) {
+            return null
+        }
+
+        @Override
+        boolean canResolveRelativePath() {
+            return false
+        }
+    }
```

Comment:
❓ Can we use a mock here instead?

Reply:
Yes, we can mock/stub it. We use a `Stub(PathToFileResolver )` now